src/Package/index.tsx

```diff
@@ -44,17 +43,85 @@ export function Package() {
   return (
     <Box w="100%">
       {/* Operator Area */}
-      <PackageHeader
-        title={scope ? `${scope}/${name}` : name}
-        description={assembly?.description}
-        tags={[]}
-      />
+      {assembly && (
+        <PackageHeader
+          title={getFullPackageName(name, scope)}
+          description={assembly?.description}
+          tags={[]}
+        />
+      )}
 
       {/* Getting Started Area */}
       <GettingStarted targets={targets} />
 
       {/* Readme and Api Reference Area */}
-      {readme && <PackageDocs readme={readme} />}
+      {assembly && (
+        <PackageDocs
+          assembly={assembly}
+          language={q.language ?? "python"}
+          submodule={q.submodule}
+        />
+      )}
     </Box>
   );
 }
+
+async function fetchAssembly(
+  name: string,
+  version: string,
+  scope?: string
+): Promise<spec.Assembly> {
+  if (version.startsWith("^")) {
+    version = version.substring(1, version.length);
+  }
+
+  // e.g https://awscdk.io/packages/@aws-cdk/alexa-ask@1.106.0/jsii.json
```

Comment:
AFAIK we can don't need to fetch from awscdk.io because we have a cloudfront proxy, no?

Reply:
Yes, debug leftovers.